api/directions/views.py

```diff
@@ -1149,29 +1149,38 @@ def directions_anesthesia_load(request):
         return JsonResponse({'data': 'Ошибка входных данных'})
     anesthesia_data = ParaclinicResult.anesthesia_value_get(research_data['iss_pk'], research_data["field_pk"])
     tb_data = []
+    row_category = {}
     if anesthesia_data:
         result = eval(anesthesia_data)
         if isinstance(result, dict):
             cols_template = [''] * (len(result['times']) + 1)
-
             times_row = ['Параметр']
             times_row.extend(result['times'])
+            times_row.append('Сумма')
 
             def made_structure(type):
                 for i in result[type]:
+                    sum = ''
                     current_param = ['' for i in cols_template]
                     current_param[0] = i
                     for k, v in result[i].items():
                         if k in times_row:
                             index = times_row.index(k)
                             current_param[index] = v
+                            if type in ['potent_drugs', 'narcotic_drugs'] and v:
+                                if sum == '':
+                                    sum = 0
+                                sum += int(v)
+                    current_param.append(sum)
```

Comment:
Если начальное значение сделать 0, то тут будет так
```suggestion
                    current_param.append(sum or '')
```

Reply:
Ну и конечно, это работает так, что если sum += int(v) прибавит ноль и в sum окажется ноль, то оно всегда не покажется (раньше в таблицу добавился бы ноль, если зашло в условие if type in ['potent_drugs', 'narcotic_drugs'] and v:)